AudioLibrary.WMME/WaveStream.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.IO;
+using AudioLibrary.WMME.Native;
+
+namespace AudioLibrary.WMME
+{
+    internal class WaveStream : Stream, IDisposable
+	{
+		private Stream m_Stream;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make 'm_Stream' 'readonly'. ([csharpsquid:S2933](https://sonarcloud.io/organizations/alex-yevtushenko3820/rules#rule_key=csharpsquid%3AS2933))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=IPP&pullRequest=7&issues=AWfG1-gy8DroMTOoScWx&open=AWfG1-gy8DroMTOoScWx)

[ISSUE_KEY]: AWfG1-gy8DroMTOoScWx